helix-term/src/commands.rs

```diff
@@ -581,21 +581,29 @@ fn goto_first_nonwhitespace(cx: &mut Context) {
 }
 
 fn goto_window(cx: &mut Context, align: Align) {
+    let count = cx.count.map(|_| cx.count() - 1);
     let (view, doc) = current!(cx.editor);
 
     let height = view.inner_area().height as usize;
 
+    // respect user given count if any
     // - 1 so we have at least one gap in the middle.
     // a height of 6 with padding of 3 on each side will keep shifting the view back and forth
     // as we type
-    let scrolloff = cx.editor.config.scrolloff.min(height.saturating_sub(1) / 2);
+    let scrolloff =
+        count.unwrap_or_else(|| cx.editor.config.scrolloff.min(height.saturating_sub(1) / 2));
```

Comment:
I think it would better to follow the vim behavior here of having count starting from scrolloff point, otherwise `1gt` would result in moving the screen by scrolloff.

Reply:
done, please take a another look